Add Component as a supported Jira field type (#1684)
semgrep/semgrep-docs: docs/semgrep-appsec-platform/jira.md

After setting up your Jira integration, you're now ready to create Jira tickets. Jira tickets can be created from findings in Semgrep Code, Supply Chain, and Secrets.

### Component Mappings

If you've created a custom field mapping for a component field type, be aware that if you choose to create tickets in a project other than your default project as configured in your integration settings, you'll need to ensure that project has access to the component you selected for your mapping.

> If your default project uses [**Jira Components**](https://support.atlassian.com/jira-software-cloud/docs/what-are-jira-components/) and you create a component field mapping in your integration settings, **you'll be unable to create tickets in another project** because Jira components are always project-scoped.

Reviewer: ```suggestion
If your default project uses [**Jira Components**](https://support.atlassian.com/jira-software-cloud/docs/what-are-jira-components/) and you create a component field mapping in your integration settings, **you cannot create tickets in another project** because Jira components are always project-scoped.
```

1. Set the status to **Open**, **Fixing**, or **Reviewing**. Select **Fixing** if it is a known issue that needs to be fixed or **Reviewing** if the finding needs more investigation.
1. Select the **Create tickets...** checkbox.
    1. Optional: Click the first drop-down list to choose between making a ticket for **groups of findings** or an individual ticket for **each finding**.

>     1. Optional: Click the **JIRA project** drop-down list to select which Jira project to add the findings to. You can choose any project that is associated with the issue type configured in your integration settings.

Author: Not related to components but think this would be good to mention here

Reviewer: SGTM!
